core-web/libs/dotcms-models/src/lib/dot-experiments-constants.ts

```diff
@@ -134,26 +134,34 @@ export const DefaultGoalConfiguration: Goals = {
     }
 };
 
-export const GOALS_METADATA_MAP: Record<GOAL_TYPES, { label: string; description: string }> = {
+export const GOALS_METADATA_MAP: Record<
+    GOAL_TYPES,
+    { label: string; description: string; icon: string }
+> = {
     [GOAL_TYPES.REACH_PAGE]: {
         label: 'experiments.goal.reach_page.name',
-        description: 'experiments.goal.reach_page.description'
+        description: 'experiments.goal.reach_page.description',
+        icon: 'pi-file-excel'
```

Comment:
Maybe you could add the missing `pi` here in this object

Reply:
lets keep just the icon for the moment.